Stratis.Bitcoin.IntegrationTests/NodeBuilder.cs

```diff
@@ -729,9 +730,10 @@ public Block[] GenerateStratis(int blockCount, List<Transaction> passedTransacti
 				blocks.Add(block);
 				if (broadcast)
 				{
-					var newChain = new ChainedBlock(block.Header, block.GetHash(), fullNode.Chain.Tip);
+                    uint256 blockHash = block.GetHash();
```

Comment:
I know I did not fix code style in those files that needed just minor update to compile. Should I fix the code style as well? If so, hopefully I am not required to add comments there ...